src/JsonSchema/Validator.php

```diff
@@ -45,10 +45,10 @@ public function check($value, $schema = null, JsonPointer $path = null, $i = nul
      *
      * {@inheritDoc}
      */
-    public function coerce(&$value, $schema = null, JsonPointer $path = null, $i = null)
+    public function coerce(&$value, $schema = null, JsonPointer $path = null, $i = null, $default = false)
```

Comment:
I was going to wait until we had some unit tests to look at, but you'll want to make sure that setting a default on `items` works, for example.